src/storekit.cs

```diff
@@ -525,34 +545,51 @@ interface SKStoreProductViewController {
 		void LoadProduct (StoreProductParameters parameters, [NullAllowed] Action<bool,NSError> callback);
 	}
 
+	[Mac (11,0), NoTV, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[Model]
+	[Protocol]
+	interface SKStoreProductViewControllerDelegate {
+		[Export ("productViewControllerDidFinish:"), EventArgs ("SKStoreProductViewController")]
+		void Finished (SKStoreProductViewController controller);
+	}
+
 	[NoWatch]
-	[NoMac]
+	[Mac (11,0)]
 	[StrongDictionary ("SKStoreProductParameterKey")]
 	interface StoreProductParameters {
 
-		[iOS (11,3), TV (11,3)]
+		[iOS (11,3), TV (11,3), NoMac]
 		[Export ("AdNetworkAttributionSignature")]
 		string AdNetworkAttributionSignature { get; set; }
 
-		[iOS (11,3), TV (11,3)]
+		[iOS (11,3), TV (11,3), NoMac]
 		[Export ("AdNetworkCampaignIdentifier")]
 		uint AdNetworkCampaignIdentifier { get; set; }
 
-		[iOS (11,3), TV (11,3)]
+		[iOS (11,3), TV (11,3), NoMac]
 		[Export ("AdNetworkIdentifier")]
 		string AdNetworkIdentifier { get; set; }
 
-		[iOS (11,3), TV (11,3)]
+		[iOS (11,3), TV (11,3), NoMac]
 		[Export ("AdNetworkNonce")]
 		NSUuid AdNetworkNonce { get; set; }
 
-		[iOS (11,3), TV (11,3)]
+		[iOS (11,3), TV (11,3), NoMac]
 		[Export ("AdNetworkTimestamp")]
 		uint AdNetworkTimestamp { get; set; }
+
+		[NoWatch, NoMac, iOS (14, 0)]
+		[Export ("AdNetworkSourceAppStoreIdentifier")]
+		string AdNetworkSourceAppStoreIdentifier { get; }
+
+		[NoWatch, NoMac, iOS (14, 0)]
+		[Export ("AdNetworkVersion")]
+		string AdNetworkVersion { get; }
```

Comment:
those should be added in `SKStoreProductParameterKey` too

Reply:
looks like they are - but code moved and the diff is much harder to read

Reply:
They where added or the code would not generate correctly.